bundle/rhpam/manifests/rhpam-kogito-operator.clusterserviceversion.yaml

```diff
@@ -61,12 +61,12 @@ metadata:
     description: RHPAM Kogito Operator for deployment and management of Kogito services.
     operators.openshift.io/valid-subscription: '["Red Hat Process Automation Manager",
       "Red Hat Decision Manager"]'
-    operators.operatorframework.io/builder: operator-sdk-v1.10.0+git
+    operators.operatorframework.io/builder: operator-sdk-v1.21.0
```

Comment:
This should not have been changed. Please use correct operator-sdk

Reply:
1.21 is the on in the README.
https://github.com/kiegroup/kogito-operator/blob/main/README.md#prerequisites

Wasn't it part of the go lang upgrade?

Reply:
oh so it has been updated ... sorry about that